pool/pool_aio_test.go

```diff
@@ -19,13 +22,43 @@ import (
 	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
 	neofscrypto "github.com/nspcc-dev/neofs-sdk-go/crypto"
 	neofsecdsa "github.com/nspcc-dev/neofs-sdk-go/crypto/ecdsa"
+	"github.com/nspcc-dev/neofs-sdk-go/eacl"
 	"github.com/nspcc-dev/neofs-sdk-go/netmap"
 	"github.com/nspcc-dev/neofs-sdk-go/object"
 	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/nspcc-dev/neofs-sdk-go/session"
 	"github.com/nspcc-dev/neofs-sdk-go/user"
+	"github.com/nspcc-dev/neofs-sdk-go/waiter"
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	defaultTimeOut = 5 * time.Second
+	tickInterval   = 1 * time.Second
+)
+
+type (
+	containerCreator interface {
+		ContainerPut(ctx context.Context, cont container.Container, prm client.PrmContainerPut) (cid.ID, error)
+	}
+
+	containerDeleter interface {
+		ContainerDelete(ctx context.Context, id cid.ID, prm client.PrmContainerDelete) error
+	}
+
+	objectPutIniter interface {
+		ObjectPutInit(ctx context.Context, hdr object.Object, prm client.PrmObjectPutInit) (*client.ObjectWriter, error)
+	}
+
+	objectDeleter interface {
+		ObjectDelete(ctx context.Context, containerID cid.ID, objectID oid.ID, prm client.PrmObjectDelete) (oid.ID, error)
```

Comment:
Not needed?

Reply:
Still needed. These interfaces are about code reusing in tests and not related to waiter functionality directly